### PR TITLE
[DNM] [WIP] Gubman 3: End of the line

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

Gubman 3 is a content PR **focused on adding a plethora of combat-related items** and changing already existing ones **for event organizers** to be able to more uniquely arm any forces they may want. Additionally, this will come with a shitton of other stuff I'm too lazy to name right now.

## Task List

- [x] Reformatting all of Cobalt's copypasted wikipedia descriptions.
- [x] Phasing out (most of) the Civ13 gun sprites for better ones.
- [ ] Overhauling and outright re-doing certain sprites.
- [ ] Implementing a plethora of firearms. Modern, futuristic and outright fictional messes.
- [ ] Implementing clothes for several un-named factions and adding a lot of choices for style.
- [ ] Phasing out silly added guns (Glocks, for one) and replacing them with stuff more appropiate to the setting.
- [ ] Melee stuff, maybe!
- [ ] James Bond gun - a suppressed Walther PPK (Maintainer request)
- [ ] More energy based weapons.
- [ ] Bring techarmor back, this time as specialized sets and not jacks of all trades.
- [ ] Bring back the Blueshield's gun beacon. Include the M45A5-E, Custom WT-550 and Custom pump-action shotgun.
- [ ] Get rid of the Head of Personnel's firearm _**(((IN FAVOR OF THE ENERGY GUN THEY USED TO GET))),**_ give the Quartermaster a unique Mosin Nagant.
- [ ] More makeshift means of defense.
- [ ] Give the Chief Engineer a viable mean of ranged defense.
_Anything more I can think of will get added here._

## Media

imagen

imagen

imagen

## Changelog
:cl:
empty for now
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/ballistic/automatic/assault_rifle/akm
-	name = "\improper Automatic Kalashnikov Rifle"
-	desc = "Introduced into service with the Soviet Army in 1959, the AKM is the prevalent variant of the entire AK series of firearms and it has found widespread use with most member states of the former Warsaw Pact and its African and Asian allies as well as being widely exported and produced in many other countries."
+	name = "\improper AKM rifle"
+	desc = "a timeless Russian infantry rifle chambered in an extremely outdated round. This is one of over two-hundred million copies produced - in and outside of Sol. A tool for a revolution, an insurgency, justice or crime. This rifle and its variants truly have no bounds."
```
Comment (tf-4) [inline modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm]:
```suggestion
	desc = "a timeless Russian infantry rifle chambered in an extremely outdated round. This is one of over two hundred million copies produced - in and outside of Sol. A tool for a revolution, an insurgency, justice, or crime. This rifle and its variants truly have no bounds."
```

File: modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm [hunk 2]
```diff
@@ -19,6 +19,7 @@
 
 /obj/item/ammo_box/magazine/akm
 	name = "akm magazine (7.62×39mm)"
+	desc = "a banana-shaped double stack magazine able to hold 30 rounds of 7.62x39mm Soviet ammunition. It's said that in the early days of SolFed's spread, spanish colony rebels often referred to these as 'Goat Horns'."
```
Comment (tf-4) [inline modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm]:
```suggestion
	desc = "a banana-shaped double-stack magazine able to hold 30 rounds of 7.62x39mm Soviet ammunition. It's said that in the early days of SolFed's spread, Spanish colony rebels often referred to these as 'Goat Horns'."
```

**Review by tf-4 — COMMENTED** [review]
tiny grammar and punctuation changes

**Comment by KasparoVy** [comment]
https://media.giphy.com/media/1eEv7v51FEI3L54jpr/giphy.gif

**Comment by KasparoVy** [comment]
We're getting GUNS
LOTS OF GUNS
guns, lots of guns

**Comment by YakumoChen** [comment]
Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.

Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.

**Comment by Draganfrukts** [comment]
> 
> 
> Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.
> 
> Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.

They have a baton and a flash. The Blueshield exists. Their office has two sets of shutters and is glued to the Bridge.

QM is in charge of the general station's budget. QM coordinates cargo technicians and miners past the HoP's "hey cook make food". The QM's firearm will not be concealable. It will be a subtype of the CC Intern Mosin.


I am not discussing this any further.

**Comment by FlamingLily** [comment]
> 
> 
> > Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.
> > Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.
> 
> They have a baton and a flash. The Blueshield exists. Their office has two sets of shutters and is glued to the Bridge.
> 
> QM is in charge of the general station's budget. QM coordinates cargo technicians and miners past the HoP's "hey cook make food". The QM's firearm will not be concealable. It will be a subtype of the CC Intern Mosin.
> 
> I am not discussing this any further.

This response feels rather hostile for no good reason

**Comment by Gandalf2k15** [comment]
> Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.
> 
> Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.

We need the command structure to be more significant. More major command changes to come.

**Comment by YakumoChen** [comment]
> > Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.
> > Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.
> 
> They have a baton and a flash. The Blueshield exists. Their office has two sets of shutters and is glued to the Bridge.
> 
> QM is in charge of the general station's budget. QM coordinates cargo technicians and miners past the HoP's "hey cook make food". The QM's firearm will not be concealable. It will be a subtype of the CC Intern Mosin.
> 
> I am not discussing this any further.

The same logic can be applied to the CE and the RD who can each make the station explode on a whim and we don't hand them M4s just because.

 The QM gets a baton and flash as well and has equal protection with the Blueshield. The QM cant spontaneously give you Armoury access however.

**Comment by FlamingLily** [comment]
> 
> 
> > > Why are you taking the HoP's gun? They have PD because they have the highest non-security access among crew.
> > > Why are you giving the QM a gun??? They don't need a gun at all, even then they could easily just use something from mining for a makeshift ranged weapon.
> > 
> > 
> > They have a baton and a flash. The Blueshield exists. Their office has two sets of shutters and is glued to the Bridge.
> > QM is in charge of the general station's budget. QM coordinates cargo technicians and miners past the HoP's "hey cook make food". The QM's firearm will not be concealable. It will be a subtype of the CC Intern Mosin.
> > I am not discussing this any further.
> 
> The same logic can be applied to the CE and the RD who can each make the station explode on a whim and we don't hand them M4s just because.
> 
> The QM gets a baton and flash as well and has equal protection with the Blueshield. The QM cant spontaneously give you Armoury access however.

The HoP's ID is effectively an all access ID on ANYONE's ID. It makes sense to give them a self defense weapon. There's a reason they're 2nd in line on CoC. What can the QM ID get you? The vault? HoP can to.

**Comment by NullDagaf** [comment]
> This response feels rather hostile for no good reason

If you had to constantly argue why your changes are good when people constantly argue that they're bad without even checking the changes nor the reasoning, you'd probably get hostile yourself.

**Comment by NullDagaf** [comment]
> Reformatting all of Cobalt's copypasted wikipedia descriptions.

Also, based.

**Comment by Gandalf2k15** [comment]
HURRY UP!!

File: modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/ballistic/automatic/assault_rifle/akm
-	name = "\improper Automatic Kalashnikov Rifle"
-	desc = "Introduced into service with the Soviet Army in 1959, the AKM is the prevalent variant of the entire AK series of firearms and it has found widespread use with most member states of the former Warsaw Pact and its African and Asian allies as well as being widely exported and produced in many other countries."
+	name = "\improper AKM rifle"
+	desc = "a timeless Russian infantry rifle chambered in an extremely outdated round. This is one of over two hundred million copies produced - in and outside of Sol. A tool for a revolution, an insurgency, justice, or crime. This rifle and its variants truly have no bounds."
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm]:
```suggestion
	desc = "A timeless Russian infantry rifle chambered in an extremely outdated round. This is one of over two hundred million copies produced - in and outside of Sol. A tool for a revolution, an insurgency, justice, or crime. This rifle and its variants truly have no bounds."
```

GRAMMAR DUDE

**Comment by Draganfrukts** [comment]
> HURRY UP!!

It will take over a month lol

**Comment by Gandalf2k15** [comment]
Re-open once yer gonna start work again :)

**Pull request closed without merging**